### DISCUSSION TITLE
Issues with neovim visual

### DISCUSSION THREAD

**Discussion by jcovington16** [LazyVim/LazyVim#2040] [Q&A]
<img width="1680" alt="Screenshot 2023-11-18 at 10 59 14 PM" src="https://github.com/LazyVim/LazyVim/assets/32824106/23997ed3-9878-422d-b87c-83c6b3bfd021">

I have an image above that shows what my installation looks like. Unfortunately it looks nothing like the default. I believe it's because I'm using ohmyzsh as my default terminal. Could I please get some assistance with this? I'm not sure what the issue is or what I'm doing wrong. Thank you guys.

**Comment by zwyyy456** [comment]
You need install nerd-font.

**Reply by jcovington16** [reply]
install nerd-font from brew? Which file within the lua directory would I ensure it's installed in?

**Comment by dimas-cyriaco** [comment]
Looks like there's something wrong with your terminal colors too. I strongly recommend you to use other terminal app. The default one on Mac is pretty bad. Go with Kitty or Alacritty.

**Reply by jcovington16** [reply]
So I should do away ohmyzsh and just use the basic terminal that comes default with a mac?

**Reply by dimas-cyriaco** [reply]
I think you are confusing some concepts. `ohmyzsh` is not a terminal, its a zsh framework. And, by the screenshot you posted, you **are** using the basic MacOS Terminal app. I don't believe `ohmyzsh` is the problem. I suggest you try other terminal app, like [Kitty](https://sw.kovidgoyal.net/kitty/), before changing your config.

**Reply by jcovington16** [reply]
Ok understood. I do appreciate the clarity. I was very confused when I installed lazyvim on my machine and saw that it didn't look the same as the default install. I took a look at different youtube videos and they all were using the basic MacOS terminal app. I will definitely give Kitty a shot and then if that doesn't work I'll reach back out to you regarding changing configs if that's ok with you?

**Reply by madflow** [reply]
Another Option is "Wezterm", which has the Nerd-Fonts baked in (https://wezfurlong.org/wezterm/config/fonts.html)

**Reply by jcovington16** [reply]
So is the basic terminal that comes with Mac not a good one to use?

**Comment by nipuna-perera** [comment]
I wonder what you saw was iTerm2. That's is as ubiquitous on Mac as the
default terminal. ITerm2 is also a good alternative if you want to
configure via UI and not code.

**Reply by jcovington16** [reply]
It probably was iterm2. I will give that a go before I do any other terminal application. I'll reach back out here in a sec.

**Reply by jcovington16** [reply]
Yea iterm2 was the best option. I'm all good now. It was definitely not a good option to use the default terminal for Mac.
